server/engine/src/main/java/org/eclipse/lsp/cobol/core/model/extendedapi/InlinePerform.java

```diff
@@ -21,10 +21,10 @@
 @Value
 @EqualsAndHashCode(callSuper = true)
 public class InlinePerform extends CFASTNode {
-  boolean untilExit;
+  PerformUntilType performUntilType;
```

Comment:
Inline perform may not have UNTIL clause and loop, so it must be reflected in the CCF